master/getting-started/kubernetes/installation/app-layer-policy.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Installing Calico with Application Layer Policy
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/app-layer-policy'
+---
+
```

Comment:
We are trying to break our lines around 80 characters if you can do that, I know some people will be happy.